extensions/wikia/GlobalNavigation/scripts/GoogleSearchAPI.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * We AB test for wikia search (both within and all wikias) in Oasis with Google custom search result.
+ * window.wgGoogleSearchTest global variable is set to true for 50% of users who visit ja wikia.
+ */
+$(function () {
+	if ( window.wgGoogleSearchTest ) {
+		var searchForm = document.getElementById('searchForm'),
+			searchId = window.wgGoogleSearchParam,
```

Comment:
for extra clarity, maybe add a comment here that this variable is set in Optimizely